library/core/class.request.php

```diff
@@ -435,7 +435,9 @@ protected function _LoadEnvironment() {
             SafeParseStr($Get, $Get, $Original);
          }
 
-         if (isset($Get['_p'])) {
+         if (isset($_SERVER['X_REWRITE']) && isset($_SERVER['PATH_INFO'])) {
```

Comment:
Is `isset($_SERVER['PATH_INFO'])` always set? I'm thinking of the case where you are on the server root. Maybe we can just check for `isset($_SERVER['X_REWRITE'])`.